### PR TITLE
Feature/user dom connection

### PR DESCRIPTION
**Pull request by jfogiato**
Files Changed: index.html, styles.css, scripts.js

Change Summary: This PR will enable the user to view their saved recipes, and then filter and search through said recipes. It will also add some slight styling and fix a few style-related bugs (such as the white space when clicking into a modal from the savedRecipes section). It will also enable going back to the home page via clicking on the logo in the top left. 

Other Notes: Super strong work from the team! A handful of follow up issues from this branch that have been logged as issues. Ready to move onto fetching data!

### CODE REVIEW

**Comment by qrispi** [comment]
Live, laugh, loving this!

Closes #61, Closes #58, Closes #57, Closes #45, Closes #42, Closes #41, Closes #38, Closes #37, Closes #36, Closes #35, Closes #34, Closes #33, Closes #32, Closes #20, Closes #9

File: dist/index.html [hunk 1]
```diff
@@ -9,8 +9,7 @@
     <nav class="nav-bar">
         <img class="logo" id="logo" src="./images/turing-logo.png" alt="Turing Logo placeholder">
         <menu class="nav-links">
-            <li id="nav-search">Search Recipes</li>
-            <input type="text" placeholder="Search.." id="searchBar">
+            <input type="text" placeholder="Search Recipes.." id="searchBar">
```
Comment (qrispi) [inline dist/index.html:12]: Still want to add a button to process the search here!

File: dist/index.html [hunk 2]
```diff
@@ -23,8 +22,8 @@
                     <li>Side Dish</li>
                  </div>
             </div>
-            <li id="nav-my-recipes">My Recipes</li>
-            <li id="nav-user-info">User Info</li>
+            <li id="navMyRecipes">My Recipes</li>
+            <li id="navUserInfo">User Info</li>
```
Comment (qrispi) [inline dist/index.html:26]: Thanks for fixing these to camelCase

File: src/scripts.js [hunk 3]
```diff
@@ -1,62 +1,81 @@
 import './styles.css';
 // import apiCalls from './apiCalls';
-// An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png';
 import './images/heart.png';
 import Recipe from './classes/Recipe';
```
Comment (qrispi) [inline src/scripts.js:5]: Are we using this? And do we need it?

**Review by jsahim — COMMENTED** [review]
Let's also not forget to update the heart graphic on this.

File: dist/index.html (hunk 1 above)
Comment (LordSchwifty) [reply dist/index.html:12]: yea looks good!

File: src/styles.css [hunk 4]
```diff
@@ -18,8 +18,14 @@ body {
   z-index: 3;
 }
 
-.direction-list {
+.nav-links {
   list-style: none;
+  display: flex;
+  width: 50%;
```
Comment (qrispi) [inline src/styles.css:24]: I think this needs to controlled by media queries once we get there.

**Review by qrispi — APPROVED** [review]

File: src/scripts.js (hunk 3 above)
Comment (jfogiato) [reply src/scripts.js:5]: No and no

**Comment by qrispi** [comment]
> Let's also not forget to update the heart graphic on this.



> 

Any suggestions?

File: src/styles.css [hunk 5]
```diff
@@ -1,9 +1,9 @@
 body {
   padding-top: 15vmin;
   text-align: center;
-  display: flex;
```
Comment (LordSchwifty) [inline src/styles.css:4]: This came out looking much better!

**Review by LordSchwifty — APPROVED** [review]
Everything looks in order on this pr!

File: src/scripts.js [hunk 6]
```diff
@@ -89,24 +108,39 @@ function toggleHidden(element) {
...
+  currentUser = new User(usersData[Math.floor(Math.random() * usersData.length)]);
+  navUserInfo.innerText = currentUser.name;
+}
+
+function showSavedRecipes() {
+  savedView = true;
+  createRecipeCards(currentUser.savedRecipes);
+}
+
+function goHome() {
```
Comment (jsahim) [inline src/scripts.js:143]: We might want to consider adding an All Recipes button to Nav that calls this function

**Review by jsahim — APPROVED** [review]
Looks good!

**Pull request merged**